Simulator/Core/Core.cpp

```diff
@@ -55,15 +55,15 @@ bool Core::parseCommandLine(string executableName, string cmdLineArguments)
    if ((cl.addParam("configfile", 'c', ParamContainer::filename, "parameter configuration filepath")
         != ParamContainer::errOk)
 #if defined(USE_GPU)
-       || (cl.addParam("device", 'd', ParamContainer::regular, "CUDA device id")
+       || (cl.addParam("device", 'g', ParamContainer::regular, "CUDA device id")
            != ParamContainer::errOk)
-#endif   // USE_GPU
-       || (cl.addParam("stimulusfile", 's', ParamContainer::filename, "stimulus input filepath")
+#endif   // USE_GPU changed 'd' to 'g' option
```

Comment:
I don't think it's necessary to have a note about the change in the comment here.